### PR TITLE
Ubuntu 20.04

### PR DESCRIPTION
**Pull request by skasperski**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # CMakeLists.txt has to be located in the project folder and cmake has to be
 # executed from 'project/build' with 'cmake ../'.
-cmake_minimum_required(VERSION 2.6)
-project(envire_core)
-
+cmake_minimum_required(VERSION 3.9)
+project(envire_core VERSION 0.1 DESCRIPTION "blablablubb")
```
Comment (m0rsch1) [inline CMakeLists.txt:4]: Maybe we should insert here a meaningfull description?

**Comment by m0rsch1** [comment]
What I have not tested yet: does this compile on Ubuntu 18.04 as well?

**Comment by skasperski** [comment]
I am currently looking into the boost-thread dependency. Quite a few other packages are affected by this as well, and I think it needs to be handled somehow differently.

**Comment by planthaber** [comment]
There is not a single QT4 dependency in this lib, so why should it be searched in the cmakelists?

**Pull request merged**